Proton/Sources/Swift/PlatformBridges/Common/TypeAliases.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  TypeAliases.swift
+//  TypeAliases
+//
+//  Created by Michał Śmiałko on 03/08/2021.
+//
+
+import Foundation
+
+#if os(iOS)
+import UIKit
+public typealias PlatformColor = UIColor
```

Comment:
I wasn't sure about the naming for a while and so it's partially random at this moment. But as I was working on that, I now want to follow a following convention:
* `Native...` is always a typealias 
* `Platform...` can be a typealias to `Native...` but on some platform it can also be a custom subclass of that type. That subclass might be needed in order to add additional logic.

For example, `PlatformView` on macOS adds `layoutSubviews` method that is called from `layout()`. Thanks to that, subclasses don't need to override `layout` on macOS and `layoutSubviews` on iOS. 

We should strive to use `Native...` as much as possible because it simpler but sometimes using `Platform...` will be inevitable.